src/mono/sample/wasm/browser/main.js

```diff
@@ -16,6 +14,12 @@ async function loadRuntime() {
     return globalThis.exports.createDotnetRuntime;
 }
 
+const anchorTagForAbsoluteUrlConversions = document.createElement('a');
```

Comment:
we can try to make something like this part of the runtime, not to ask all applications to do it themselves.
but note that nodejs don't have DOM, so there must be another way.